<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>main</class>
 <widget class="QWidget" name="main">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>540</width>
    <height>750</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>540</width>
    <height>750</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
	border-radius : &quot;10px&quot;;
}</string>
  </property>
  <widget class="QLabel" name="bgLbl">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>540</width>
     <height>750</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>540</width>
     <height>750</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(47, 47, 47);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="logoLbl">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>0</y>
     <width>355</width>
     <height>122</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	image: url(:/5 1.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="fullScreenBtn">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>20</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    border-image: url(:/image 27.png);
    background-color : transparent;
}

QPushButton:hover {
    background-color: rgba(0,0,0,0.2);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="filterBtn">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>20</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    
	image: url(:/image 5.png);
    background-color : transparent;
}

QPushButton:hover {
    background-color: rgba(0,0,0,0.2);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="closeBtn">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>25</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    
	border-image: url(:/X.png);
    background-color : transparent;
}

QPushButton:hover {
    background-color: rgba(0,0,0,0.2);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="recordBtn">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>599</y>
     <width>35</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	
	border-image: url(:/image 3.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>590</y>
     <width>427</width>
     <height>53</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	background-color : &quot;#666666&quot;;
	border-radius : &quot;18px&quot;;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="messagInput">
   <property name="geometry">
    <rect>
     <x>86</x>
     <y>591</y>
     <width>341</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
	background-color : &quot;#666666&quot;;
	color : &quot;#CBCBCB&quot;;
}</string>
   </property>
   <property name="placeholderText">
    <string>Enter message</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sendMsgBtn">
   <property name="geometry">
    <rect>
     <x>441</x>
     <y>596</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	background-color : &quot;#FFFFFF&quot;;
	border-image: url(:/image 4.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="noteLbl">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>670</y>
     <width>130</width>
     <height>62</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	background-color : &quot;#666666&quot;;
	border-radius : &quot;22px&quot;;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="captureBtn">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>670</y>
     <width>130</width>
     <height>62</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	background-color : &quot;#666666&quot;;
	border-radius : &quot;22px&quot;;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="smodeLbl">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>670</y>
     <width>130</width>
     <height>62</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	background-color : &quot;#666666&quot;;
	border-radius : &quot;22px&quot;;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="noteBtn">
   <property name="geometry">
    <rect>
     <x>77</x>
     <y>678</y>
     <width>114</width>
     <height>45</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	color : &quot;#FFFFFF&quot;;
}</string>
   </property>
   <property name="text">
    <string>Note</string>
   </property>
   <property name="icon">
    <iconset resource="../resource/resource.qrc">
     <normaloff>:/image 6.png</normaloff>:/image 6.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="smodeBtn">
   <property name="geometry">
    <rect>
     <x>227</x>
     <y>678</y>
     <width>115</width>
     <height>44</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	color : &quot;#FFFFFF&quot;;
}</string>
   </property>
   <property name="text">
    <string>Speech Mode</string>
   </property>
   <property name="icon">
    <iconset resource="../resource/resource.qrc">
     <normaloff>:/image 7.png</normaloff>:/image 7.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="smodeBtn_2">
   <property name="geometry">
    <rect>
     <x>378</x>
     <y>677</y>
     <width>113</width>
     <height>48</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	color : &quot;#FFFFFF&quot;;
}</string>
   </property>
   <property name="text">
    <string>Screenshot</string>
   </property>
   <property name="icon">
    <iconset resource="../resource/resource.qrc">
     <normaloff>:/image 8.png</normaloff>:/image 8.png</iconset>
   </property>
  </widget>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>130</y>
     <width>481</width>
     <height>451</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(47,47,47);</string>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAsNeeded</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>481</width>
      <height>451</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../resource/resource.qrc"/>
 </resources>
 <connections/>
</ui>
